Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -107,6 +120,11 @@ class TextureCacheBase
     u32 GetNumLevels() const { return texture->GetConfig().levels; }
     u32 GetNumLayers() const { return texture->GetConfig().layers; }
     AbstractTextureFormat GetFormat() const { return texture->GetConfig().format; }
+    u32 InMemoryFormat() const { return format & 0xff; }
```

Comment:
Part of the original reason for this change was to allow easy changing of the mask in just one place. 

My version of hybrid introduced an in memory format which was 6 bits long. I'm not sure what the current status of that is.